cluster/saltbase/salt/rescheduler/rescheduler.manifest

```diff
@@ -28,8 +28,8 @@ spec:
     # TODO: split this out into args when we no longer need to pipe stdout to a file #6428
     - sh
     - -c
-    - '/rescheduler --running-in-cluster=false 1>>/var/log/rescheduler.log 2>&1'
+    - '/rescheduler --running-in-cluster=false --log_dir=/var/log/rescheduler --logtostderr=false'
```

Comment:
As this is belong to salt, seems the salt will create rescheduler.log, please check here https://github.com/kubernetes/kubernetes/blob/master/cluster/saltbase/salt/rescheduler/init.sls#L11